packages/arb-token-bridge-ui/src/util/RetryableUtils.ts

```diff
@@ -1,23 +1,26 @@
 import {
   L1TransactionReceipt,
-  L1ToL2MessageWriter as IL1ToL2MessageWriter
+  L1ToL2MessageWriter as IL1ToL2MessageWriter,
+  L1ToL2MessageStatus
 } from '@arbitrum/sdk'
 import { Signer } from '@ethersproject/abstract-signer'
 import { Provider } from '@ethersproject/abstract-provider'
 import dayjs from 'dayjs'
 import { JsonRpcProvider } from '@ethersproject/providers'
+import { MergedTransaction } from '../state/app/state'
+import { isTeleport } from '../token-bridge-sdk/teleport'
 
 type GetRetryableTicketParams = {
-  l1TxHash: string
+  sourceChainTxHash: string
   retryableCreationId?: string
-  l1Provider: Provider
-  l2Signer: Signer
+  sourceChainProvider: Provider
+  destinationChainSigner: Signer
 }
 
 type GetRetryableTicketExpirationParams = {
-  l1TxHash: string
-  l1Provider: JsonRpcProvider
-  l2Provider: JsonRpcProvider
+  sourceChainTxHash: string
+  sourceChainProvider: JsonRpcProvider
+  destinationChainProvider: JsonRpcProvider
 }
```

Comment:
as mentioned on other review comments, I think `parent` and `child` would be better here

Reply:
Updated